TKGame/TKGame.cs

```diff
@@ -1,6 +1,8 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using Microsoft.Xna.Framework.Input;
+using Myra;
```

Comment:
Might want to include a comment to describe what Myra is for those who haven't seen it before

Reply:
Fair, I have a small comment in Debug.cs, but I can move it to the main file